modules/models/src/main/java/org/eclipse/xpanse/modules/models/serviceconfiguration/exceptions/ServiceConfigurationNotValidException.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.serviceconfiguration.exceptions;
+
+/**
+ * Exception thrown when service configuration is invalid.
+ */
+
+public class ServiceConfigurationNotValidException extends RuntimeException {
```

Comment:
It is better to rename it to 'ServiceConfigurationInvalidException' for maintaining a consistent style with other custom exceptions